GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -76,6 +76,8 @@ public sealed partial class FormBrowse : GitModuleForm, IBrowseRepo
         private readonly TranslationString _undoLastCommitText = new TranslationString("You will still be able to find all the commit's changes in the staging area\n\nDo you want to continue?");
         private readonly TranslationString _undoLastCommitCaption = new TranslationString("Undo last commit");
 
+        private readonly TranslationString _runShellFailed = new TranslationString("Fail to run the shell. Reason: ");
```

Comment:
"Failed to ..." or "Failure to ..."?
```suggestion
        private readonly TranslationString _runShellFailed = new TranslationString("Failed to run the shell. Reason:");
```

Reply:
Suggest telling which shell failed too:
```suggestion
        private readonly TranslationString _runShellFailed = new TranslationString("Failed to run '{0}' shell.{1}Reason: {2}");
```